src/redux/actions/samples/deleteSamples.js

```diff
@@ -87,26 +105,36 @@ const deleteSamples = (
         // This is set right now as there is only one experiment per project
         // Should be changed when we support multiple experiments per project
         const experimentId = projects[projectUuid].experiments[0];
-        await sendDeleteSamplesRequest(projectUuid, experimentId, sampleUuids);
-
-        dispatch(saveProject(projectUuid, newProject, false));
 
-        dispatch({
-          type: PROJECTS_UPDATE,
-          payload: {
-            projectUuid,
-            project: {
-              samples: newSamples,
-            },
-          },
-        });
+        if (config.currentApiVersion === api.V1) {
+          await sendDeleteSamplesRequest(projectUuid, experimentId, sampleUuids);
 
-        dispatch({
-          type: SAMPLES_DELETE,
-          payload: { sampleUuids: samplesToDelete },
-        });
+          dispatch(saveProject(projectUuid, newProject, false));
 
-        dispatch(updateExperiment(experimentId, { sampleIds: newSamples }));
+          dispatch({
+            type: PROJECTS_UPDATE,
+            payload: {
+              projectUuid,
+              project: {
+                samples: newSamples,
+              },
+            },
+          });
+
+          dispatch({
+            type: SAMPLES_DELETE_API_V1,
```

Comment:
it seems that both SAMPLES_DELETE_API_V1 and SAMPLES_DELETE use the same reducer. Then why we need to separate action types?